docker-compose.yml

```diff
@@ -44,6 +44,7 @@ services:
       - BB_CLIENT_CERT_FILE=../shared_files/bb-dev-test-cert.pem
       - BB_CLIENT_KEY_FILE=../shared_files/bb-dev-test-key.pem
       - BB_SERVER_LOCATION=https://fhir.backend.bluebutton.hhsdevcloud.us
+      - BB_CHECK_CERT=false
```

Comment:
is BB_CHECK_CERT needed for both worker and api or just the worker container? Note: I know nothing about docker

Reply:
You're right! Should be worker only.